app/Http/Controllers/Admin/DashboardController.php

```diff
@@ -4,16 +4,28 @@
 
 use App\Model\Post;
 use App\Http\Controllers\Controller;
+use App\Repository\PostRepository;
 use Illuminate\View\View;
 
 class DashboardController extends Controller
 {
+    /**
+     * @var PostRepository
+     */
+    private $postRepository;
+
+    public function __construct(PostRepository $postRepository)
+    {
+        $this->postRepository = $postRepository;
+    }
 
     /**
      * @return \Illuminate\Contracts\View\Factory|View
      */
     public function index(): View
     {
-        return view('admin.dashboard.index', ['posts_count' => Post::count()]);
+        $postCount = $this->postRepository->count();
```

Comment:
Tu peux peut être mettre directement dans le tableau le `$this->postRepository->count()` on gagne une ligne, je ne sais pas ce que tu en pense ?

Reply:
@TuxBoy Je trouve que cela apporte de la lisibilité et rend la maintenance plus simple.